src/router/index.js

```diff
@@ -27,7 +27,6 @@ const routes = [
       // legacy URLs support redirectID, new URLs must use redirectId
       redirectId: route.query.redirectID || route.query.redirectId,
       group: route.query.group,
-      authType: route.query.authType,
```

Comment:
why was this removed?
because the authType will be available in session/group data is that why?
If yes, then what about the older links which had `authType` as param? Will they be working fine?

Reply:
Yeah, and also, I had a fallback value in the code. 

![Screenshot 2022-11-16 at 11 46 58 AM](https://user-images.githubusercontent.com/25496389/202099291-5ba54570-b67a-49d5-8d86-1a7d7da9a27b.png)